filebeat/filebeat.yml

```diff
@@ -151,6 +151,9 @@ filebeat:
   # Event count spool threshold - forces network flush if exceeded
   #spool_size: 2048
 
+  # Enable async publisher pipeline in filebeat (Experimental!)
+  #publish_async: false
```

Comment:
This will probably disappear after rebase: https://github.com/elastic/beats/blob/master/filebeat/filebeat.yml#L155